utils/clients/tenderly.ts

```diff
@@ -313,10 +311,17 @@ export async function simulateNew(config: SimulationConfigNew): Promise<Simulati
       ...simulationPayload.state_objects[from],
       balance: totalValue.toString(),
     };
+
+    // Also ensure the timelock has enough ETH to execute the proposal
+    simulationPayload.state_objects[timelock.address] = {
```

Comment:
need to think more about out if this is kosher to just override the timelock's balance